### ISSUE TITLE
Version 4.1.2 is having problems with detecting branch in github actions

### ISSUE THREAD

**Issue by avizcaino** [pvdlg/env-ci#103]
## Bug

Last release (4.1.2) is having problems when releasing from Github Actions. It seems that it's unable to detect which branch is on:

```
2019-09-27T09:39:08.8434165Z ##[group]Run npm run semantic-release -- --debug
2019-09-27T09:39:08.8434411Z [36;1mnpm run semantic-release -- --debug[0m
2019-09-27T09:39:08.8492343Z shell: /bin/bash -e {0}
2019-09-27T09:39:08.8492529Z env:
2019-09-27T09:39:08.8492937Z   GITHUB_TOKEN: ***
2019-09-27T09:39:08.8493466Z   NPM_TOKEN: ***
2019-09-27T09:39:08.8493613Z ##[endgroup]
2019-09-27T09:39:09.1890501Z 
2019-09-27T09:39:09.1894094Z > @uxland/generator-lit-component@0.0.0-development semantic-release /home/runner/work/generator-lit-component/generator-lit-component
2019-09-27T09:39:09.1895338Z > semantic-release "--debug"
2019-09-27T09:39:09.1895732Z 
2019-09-27T09:39:09.7256555Z [9:39:09 AM] [semantic-release] › ℹ  Running semantic-release version 16.0.0-beta.24
2019-09-27T09:39:09.7516998Z 2019-09-27T09:39:09.751Z semantic-release:config load config from: /home/runner/work/generator-lit-component/generator-lit-component/.releaserc.json
2019-09-27T09:39:09.7582194Z 2019-09-27T09:39:09.757Z semantic-release:config options values: { branches:
2019-09-27T09:39:09.7583573Z    [ '+([1-9])?(.{+([1-9]),x}).x',
2019-09-27T09:39:09.7584305Z      'master',
2019-09-27T09:39:09.7585490Z      'next',
2019-09-27T09:39:09.7585839Z      'next-major',
2019-09-27T09:39:09.7586190Z      { name: 'beta', prerelease: true },
2019-09-27T09:39:09.7586838Z      { name: 'alpha', prerelease: true } ],
2019-09-27T09:39:09.7587576Z   repositoryUrl: 'git+https://github.com/uxland/generator-lit-component.git',
2019-09-27T09:39:09.7587912Z   tagFormat: 'v${version}',
2019-09-27T09:39:09.7588094Z   plugins:
2019-09-27T09:39:09.7588425Z    [ '@semantic-release/commit-analyzer',
2019-09-27T09:39:09.7588774Z      '@semantic-release/release-notes-generator',
2019-09-27T09:39:09.7589092Z      '@semantic-release/npm',
2019-09-27T09:39:09.7589467Z      '@semantic-release/github' ],
2019-09-27T09:39:09.7589618Z   _: [],
2019-09-27T09:39:09.7589920Z   debug: true,
2019-09-27T09:39:09.7590249Z   '$0': 'node_modules/.bin/semantic-release' }
2019-09-27T09:39:10.6617162Z 2019-09-27T09:39:10.013Z semantic-release:plugins options for @semantic-release/npm/verifyConditions: {}
2019-09-27T09:39:10.6617970Z [9:39:10 AM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/npm"
2019-09-27T09:39:10.6620401Z 2019-09-27T09:39:10.014Z semantic-release:plugins options for @semantic-release/github/verifyConditions: {}
2019-09-27T09:39:10.6620856Z [9:39:10 AM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/github"
2019-09-27T09:39:10.6621900Z 2019-09-27T09:39:10.015Z semantic-release:plugins options for @semantic-release/commit-analyzer/analyzeCommits: {}
2019-09-27T09:39:10.6622339Z [9:39:10 AM] [semantic-release] › ✔  Loaded plugin "analyzeCommits" from "@semantic-release/commit-analyzer"
2019-09-27T09:39:10.6623847Z 2019-09-27T09:39:10.015Z semantic-release:plugins options for @semantic-release/release-notes-generator/generateNotes: {}
2019-09-27T09:39:10.6624406Z 2019-09-27T09:39:10.015Z semantic-release:plugins options for @semantic-release/npm/prepare: {}
2019-09-27T09:39:10.6624907Z 2019-09-27T09:39:10.016Z semantic-release:plugins options for @semantic-release/npm/publish: {}
2019-09-27T09:39:10.6625362Z 2019-09-27T09:39:10.016Z semantic-release:plugins options for @semantic-release/github/publish: {}
2019-09-27T09:39:10.6625815Z 2019-09-27T09:39:10.016Z semantic-release:plugins options for @semantic-release/npm/addChannel: {}
2019-09-27T09:39:10.6626270Z 2019-09-27T09:39:10.021Z semantic-release:plugins options for @semantic-release/github/addChannel: {}
2019-09-27T09:39:10.6627184Z 2019-09-27T09:39:10.021Z semantic-release:plugins options for @semantic-release/github/success: {}
2019-09-27T09:39:10.6627558Z 2019-09-27T09:39:10.022Z semantic-release:plugins options for @semantic-release/github/fail: {}
2019-09-27T09:39:10.6628458Z [9:39:10 AM] [semantic-release] › ✔  Loaded plugin "generateNotes" from "@semantic-release/release-notes-generator"
2019-09-27T09:39:10.6629674Z 2019-09-27T09:39:10.492Z semantic-release:get-tags found tags: [ { gitTag: 'v1.0.0-alpha.1@alpha', version: '1.0.0-alpha.1', channel: 'alpha' } ]
2019-09-27T09:39:10.6631192Z 2019-09-27T09:39:10.510Z semantic-release:get-tags found tags for branch master: [ { version: '1.0.0-alpha.1', channel: 'alpha', gitTag: 'v1.0.0-alpha.1@alpha', gitHead: 'ceb1268a9660a843ee71e8afdb69292c8c0269ce' } ]
2019-09-27T09:39:10.6632191Z 2019-09-27T09:39:10.530Z semantic-release:get-tags found tags for branch alpha: [ { version: '1.0.0-alpha.1', channel: 'alpha', gitTag: 'v1.0.0-alpha.1@alpha', gitHead: 'ceb1268a9660a843ee71e8afdb69292c8c0269ce' } ]
2019-09-27T09:39:10.6633549Z [9:39:10 AM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/npm"
2019-09-27T09:39:10.6634012Z [9:39:10 AM] [semantic-release] › ✔  Loaded plugin "publish" from "@semantic-release/npm"
2019-09-27T09:39:10.6634449Z [9:39:10 AM] [semantic-release] › ✔  Loaded plugin "publish" from "@semantic-release/github"
2019-09-27T09:39:10.6634893Z [9:39:10 AM] [semantic-release] › ✔  Loaded plugin "addChannel" from "@semantic-release/npm"
2019-09-27T09:39:10.6635357Z [9:39:10 AM] [semantic-release] › ✔  Loaded plugin "addChannel" from "@semantic-release/github"
2019-09-27T09:39:10.6635796Z [9:39:10 AM] [semantic-release] › ✔  Loaded plugin "success" from "@semantic-release/github"
2019-09-27T09:39:10.6636229Z [9:39:10 AM] [semantic-release] › ✔  Loaded plugin "fail" from "@semantic-release/github"
2019-09-27T09:39:10.6636909Z [9:39:10 AM] [semantic-release] › ℹ  This test run was triggered on the branch undefined, while semantic-release is configured to only publish from master, alpha, therefore a new version won’t be published.
```

- Install last `semantic-release@beta`. This includes `env-ci@4.1.2`: When releasing from **any** branch, it is unable to detect which branch is working on.
- Fix `env-ci@4.1.1`: this releases without problem
```
[9:45:39 AM] [semantic-release] › ✔  Completed step "success" of plugin "@semantic-release/github"
[9:45:39 AM] [semantic-release] › ✔  Published release 1.0.0-alpha.2
```

Environment:
- CI: Github Actions
- Build link: https://github.com/uxland/generator-lit-component/commit/c1ad419ee3dff262ca10f12b3e856b4276684ce6/checks

**Comment by vitorarantes** [comment]
same problem here, had to revert back to version 4.1.1

**Comment by avizcaino** [comment]
The problem seems to be in the last PR (#100):

![image](https://user-images.githubusercontent.com/12660226/65761653-8732cd00-e11f-11e9-943a-0d088d53cb46.png)

**Comment by coderbyheart** [comment]
Fix in #101

**Comment by pvdlg** [comment]
Fixed in #101
